index.py

```diff
@@ -59,14 +78,15 @@
 with open("config.json", "w") as f:
     # Check if 'token' key exists in the config.json file
     config["token"] = token
+
     # This dumps our working setting to the config.json file
     # Indent is used to make the file look nice and clean
     # If you don't want to indent, you can remove the indent=2 from code
     json.dump(config, f, indent=2)
 
 
 class FunnyBadge(Client):
-    def __init__(self, *, intents: Intents):
+    def __init__(self, *, intents: Intents) -> None:
```

Comment:
I mean isn't it common knowledge that init never returns anything? Correct me if I am wrong lmao

Reply:
Yeah, sorry, that's something my formatter automatically did for me. I can remove it lol